wp-admin/includes/class-core-upgrader.php

```diff
@@ -266,121 +212,66 @@ public function upgrade( $current, $args = array() ) {
-		$upgrade_dev   = get_site_option( 'auto_update_core_dev', 'enabled' ) === 'enabled';
-		$upgrade_minor = get_site_option( 'auto_update_core_minor', 'enabled' ) === 'enabled';
-		$upgrade_major = get_site_option( 'auto_update_core_major', 'unset' ) === 'enabled';
-
-		// WP_AUTO_UPDATE_CORE = true (all), 'beta', 'rc', 'development', 'branch-development', 'minor', false.
-		if ( defined( 'WP_AUTO_UPDATE_CORE' ) ) {
-			if ( false === WP_AUTO_UPDATE_CORE ) {
-				// Defaults to turned off, unless a filter allows it.
-				$upgrade_dev   = false;
-				$upgrade_minor = false;
-				$upgrade_major = false;
-			} elseif ( true === WP_AUTO_UPDATE_CORE
-				|| in_array( WP_AUTO_UPDATE_CORE, array( 'beta', 'rc', 'development', 'branch-development' ), true )
-			) {
-				// ALL updates for core.
-				$upgrade_dev   = true;
-				$upgrade_minor = true;
-				$upgrade_major = true;
-			} elseif ( 'minor' === WP_AUTO_UPDATE_CORE ) {
-				// Only minor updates for core.
-				$upgrade_dev   = false;
-				$upgrade_minor = true;
-				$upgrade_major = false;
-			}
-		}
-
-		// 1: If we're already on that version, not much point in updating?
-		if ( $offered_ver === $retraceur_version ) {
-			return false;
-		}
-
-		// 2: If we're running a newer version, that's a nope.
-		if ( version_compare( $retraceur_version, $offered_ver, '>' ) ) {
-			return false;
-		}
-
-		$failure_data = get_site_option( 'auto_core_update_failed' );
-		if ( $failure_data ) {
-			// If this was a critical update failure, cannot update.
-			if ( ! empty( $failure_data['critical'] ) ) {
-				return false;
-			}
-
-			// Don't claim we can update on update-core.php if we have a non-critical failure logged.
-			if ( $retraceur_version === $failure_data['current'] && str_contains( $offered_ver, '.1.next.minor' ) ) {
-				return false;
-			}
-
-			/*
-			 * Cannot update if we're retrying the same A to B update that caused a non-critical failure.
-			 * Some non-critical failures do allow retries, like download_failed.
-			 * 3.7.1 => 3.7.2 resulted in files_not_writable, if we are still on 3.7.1 and still trying to update to 3.7.2.
-			 */
-			if ( empty( $failure_data['retry'] ) && $retraceur_version === $failure_data['current'] && $offered_ver === $failure_data['attempted'] ) {
-				return false;
-			}
-		}
-
-		// 3: 3.7-alpha-25000 -> 3.7-alpha-25678 -> 3.7-beta1 -> 3.7-beta2.
-		if ( $current_is_development_version ) {
-
-			/**
-			 * Filters whether to enable automatic core updates for development versions.
-			 *
-			 * @since WP 3.7.0
-			 *
-			 * @param bool $upgrade_dev Whether to enable automatic updates for
-			 *                          development versions.
-			 */
-			if ( ! apply_filters( 'allow_dev_auto_core_updates', $upgrade_dev ) ) {
-				return false;
-			}
-			// Else fall through to minor + major branches below.
-		}
+		_deprecated_function( __METHOD__, '2.0.0', '', true );
+
+		$not_supported = __( 'The WP Auto Updates feature is not used by Retraceur fork.' );
```

Comment:
```suggestion
		$not_supported = __( 'The WP Background Updates feature is not used by Retraceur fork.' );
```

Seems more accurate to use Background instead of auto as downloading, installing and upgrading is **automatically** achieved when using the Retraceur Coeur Update API.